csi.proto

```diff
@@ -1494,6 +1494,11 @@ message NodeExpandVolumeRequest {
   // access_type from given volume_path for the volume and perform
   // node expansion. This is an OPTIONAL field.
   VolumeCapability volume_capability = 5;
+
+  // Secrets required by plugin to complete node expand volume request.
+  // This field is OPTIONAL. Refer to the `Secrets Requirements`
+  // section on how to use this field.
+  map<string, string> secrets = 6 [(csi_secret) = true, (alpha_field) = true];
```

Comment:
you have to split this line, for now. our tooling does not like long lines. I think tooling should be fixed to make an exception for code lines but that is a discussion for another issue perhaps.

Reply:
Could try

```
  map<string, string> secrets = 6
      [(csi_secret) = true, (alpha_field) = true];
```